tools/nl/embeddings/utils.py

```diff
@@ -55,10 +53,25 @@ def _is_gcs_path(path: str) -> bool:
   return path.strip().startswith(_GCS_PATH_PREFIX)
 
 
-def _get_gcs_parts(gcs_path: str) -> tuple[str, str]:
+def _get_gcs_parts(gcs_path: str) -> Tuple[str, str]:
   return gcs_path[len(_GCS_PATH_PREFIX):].split('/', 1)
 
 
+@dataclass
+class ModelInfo:
```

Comment:
Renamed to Config now?

Reply:
this is just specific to the embedding tool but updated to make it consistent